testsuite/features/build_validation/reposync/srv_sync_all_products.feature

```diff
@@ -188,6 +188,16 @@ Feature: Synchronize products in the products page of the Setup Wizard
     When I click the Add Product button
     And I wait until I see "Ubuntu 20.04" product has been added
 
+@ubuntu2204_minion
+  Scenario: Add Ubuntu 22.04
+    When I follow the left menu "Admin > Setup Wizard > Products"
+    And I wait until I do not see "Loading" text
+    And I enter "Ubuntu 22.04" as the filtered product description
+    And I select "Ubuntu 22.04" as a product
+    Then I should see the "Ubuntu 22.04" selected
+    When I click the Add Product button
```

Comment:
I can't confirm it, as I can't see Ubuntu 22.04 in my current environment.
But we might need to select also some child channel not selected automatically, for instance something related to LTSS.
As we do in `When I select "SUSE Linux Enterprise Server LTSS 15 x86_64" as a product`

Reply:
I copied this from Ubuntu 20.04 as a wild guess. This seems to be the same for all Ubuntu and Debian variants. And Ubuntu 22.04 is a LTS release by definition.